bundles/org.openhab.binding.fritzboxtr064/README.md

```diff
@@ -0,0 +1,103 @@
+This is an optional parameter and multiple values are allowed.
+
+### `subdevice`, `subdeviceLan`
+
+Besides the bridge the thing is attached to, sub-devices have a `uuid` parameter.
+This is the UUID/UDN of the device and a mandatory parameter.
+Since the value can only be determined by examining the SCPD of the root device it is highly recommended to use discovery for adding sub-devices.
+
+For `subdeviceLan` devices (type detected automatically during discovery) a second parameter is available `macOnline`.
+It adds a channel for each MAC (format 11:11:11:11:11:11) that shows the online status of the respective device.
+This is an optional parameter and multiple values are allowed.
+
+## Channels
+
+| channel | item-type | description |
+|----|----|----|
+| `callDeflectionEnable` | `Switch`| Enable/Disable the call deflection setup with the given index. |
+| `deviceLog` | `String`| A string containing the last log messages. |
+| `dslCRCErrors` | `Number:Dimensionless`| DSL CRC Errors |
+| `dslDownstreamNoiseMargin` | `Number:Dimensionless`| DSL Downstream Noise Margin |
+| `dslDownstreamNoiseMargin` | `Number:Dimensionless`| DSL Downstream Attenuation |
+| `dslEnable` | `Switch`| DSL Enable |
+| `dslFECErrors` | `Number:Dimensionless`| DSL FEC Errors |
+| `dslHECErrors` | `Number:Dimensionless`| DSL HEC Errors |
+| `dslStatus` | `Switch`| DSL Status |
+| `dslUpstreamNoiseMargin` | `Number:Dimensionless`| DSL Upstream Noise Margin |
+| `dslUpstreamNoiseMargin` | `Number:Dimensionless`| DSL Upstream Attenuation |
+| `inboundCalls` | `Number`| Number of inbound calls within the given number of days. |
+| `macOnline` | `Switch`| Online status of the device with the given MAC |
+| `missedCalls` | `Number`| Number of missed calls within the given number of days. |
+| `outboundCalls` | `Number`| Number of outbound calls within the given number of days. |
+| `reboot` | `Switch`| Reboot |
+| `rejectedCalls` | `Number`| Number of rejected calls within the given number of days. |
+| `securityPort` | `Number`| The port for connecting via HTTPS to the TR-064 service. |
+| `tamEnable` | `Switch`| Enable/Disable the answering machine with the given index. |
+| `tamNewMessages` | `Number`| The number of new messages of the given answering machine. |
+| `uptime` | `Number:Time`| Uptime |
+| `uptime` | `Number:Time`| Uptime |
+| `wanAccessType` | `String`| Access Type |
+| `wanConnectionStatus` | `String`| Connection Status |
+| `wanIpAddress` | `String`| WAN IP Address |
+| `wanMaxDownstreamRate` | `Number:DataTransferRate`| Max. Downstream Rate |
+| `wanMaxUpstreamRate` | `Number:DataTransferRate`| Max. Upstream Rate |
+| `wanPhysicalLinkStatus` | `String`| Link Status |
+| `wanTotalBytesReceived` | `Number:DataAmount`| Total Bytes Received |
+| `wanTotalBytesSent` | `Number:DataAmount`| Total Bytes Send |
+| `wifi24GHzEnable` | `Switch`| Enable/Disable the 2.4 GHz WiFi device. |
```

Comment:
Might it make sense to offer a convenience channel `wifiEnabled`, which switches whatever wifi networks are available (2.4, 5, guest)? Imho, there are users that want to turn of wifi in general at night or while they are away. Having a single switch channel for that would probably be useful for such cases.